apps/ts/automation/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
this must be referenced in https://github.com/hmcts/cnp-flux-config/blob/master/apps/flux-system/automation/kustomization.yaml

Reply:
Just done. Thanks.